libraries/AP_ESC_Telem/AP_ESC_Telem.h

```diff
@@ -0,0 +1,32 @@
+#pragma once
+
+#include <AP_HAL/AP_HAL.h>
+
+class AP_ESC_Telem {
+public:
+
+    AP_ESC_Telem();
+
+    /* Do not allow copies */
+    AP_ESC_Telem(const AP_ESC_Telem &other) = delete;
+    AP_ESC_Telem &operator=(const AP_ESC_Telem&) = delete;
+
+    static AP_ESC_Telem *get_singleton();
+
+    // methods to determine which ESC telemetry types are active
+    bool tcan_is_active() const;
+    bool kdecan_is_active() const;
+    bool uavcan_is_active() const;
+
+    // get an individual ESC's usage time in seconds if available, returns true on success
+    bool get_usage(uint8_t esc_id, uint32_t& usage_sec) const;
```

Comment:
`get_usage_seconds` maybe? The name by itself is not enlightening at all as to what the function gives you...

Reply:
sure, renamed to get_usage_seconds()